src/Dev/SapphireTest.php

```diff
@@ -402,6 +414,12 @@ public function setUpOnce()
         }
     }
 
+    public static function tearDownAfterClass()
+    {
+        self::$test_singleton->tearDownOnce();
```

Comment:
Nope, we can't use singleton because `setUpOnce` stores state on the object instance so we need to keep a reference to the object instance.